code/modules/clothing/suits/hooded_vr.dm

```diff
@@ -11,3 +11,60 @@
 	hoodtype = /obj/item/clothing/head/hood/techpriest
 	armor = list(melee = 20, bullet = 10, laser = 10, energy = 10, bomb = 25, bio = 50, rad = 25)
 	item_state_slots = list(slot_r_hand_str = "engspace_suit", slot_l_hand_str = "engspace_suit")
+
+// Regular armor versions here, costumes below
+/obj/item/clothing/suit/storage/hooded/knight
+	name = "crusader's armor"
+	desc = "ye olde knight, risen again."
+	icon_state = "galahad"
+	icon = 'icons/obj/clothing/knights_vr.dmi'
+	icon_override = 'icons/obj/clothing/knights_vr.dmi'
+	hoodtype = /obj/item/clothing/head/hood/galahad
+	armor = list(melee = 80, bullet = 50, laser = 10, energy = 0, bomb = 0, bio = 0, rad = 0)
+	siemens_coefficient = 2
+	action_button_name = "Toggle Knight Headgear"
+	
+/obj/item/clothing/suit/storage/hooded/knight/galahad
+	name = "crusader's armor"
+	desc = "ye olde knight, risen again."
+	icon_state = "galahad"
+	hoodtype = /obj/item/clothing/head/hood/galahad
+	
+/obj/item/clothing/suit/storage/hooded/knight/lancelot
+	name = "crusader's armor"
+	desc = "ye olde knight, risen again."
+	icon_state = "lancelot"
+	hoodtype = /obj/item/clothing/head/hood/lancelot
+	
+/obj/item/clothing/suit/storage/hooded/knight/robin
+	name = "crusader's armor"
+	desc = "ye olde knight, risen again. This one seems slightly faster than the rest, but weaker."
+	icon_state = "robin"
+	hoodtype = /obj/item/clothing/head/hood/robin
+	armor = list(melee = 70, bullet = 40, laser = 10, energy = 0, bomb = 0, bio = 0, rad = 0)
+	slowdown = -1
```

Comment:
-1 slowdown 70 melee armor seems absolutely insane

Reply:
Ask Scree. :p

Reply:
you have to fight a 2000hp killer rabbit with a frag grenade to obtain it and it's useless anywhere without breathable atmos and provides no protection against anything sec will be using on you if you decide to fuck around and find out, in fact it should make you MORE vulnerable to tasers

Reply:
I can see it being pretty common though, considering combination of being lowest cost for the area (tied with generic vaults) and lack of discard chance.

Reply:
Single-spawn only POI, and frankly, it's also 20x20, so it'll be difficult to fit given it's only like 5-10% passthrough.